sdk/include/opentelemetry/sdk/metrics/aggregator/sketch_aggregator.h

```diff
@@ -118,7 +118,7 @@ class SketchAggregator final : public Aggregator<T>
       idx = iter->first;
       count += iter->second;
     }
-    return round(2 * pow(gamma, idx) / (gamma + 1));
+    return (T)(round(2 * pow(gamma, idx) / (gamma + 1)));
```

Comment:
Prefer `static_cast`?

Reply:
@ThomsonTan - fixed in the latest update.